Arrow chunk rows example in docs (#25)
KxSystems/arrowkdb: docs/reference.md

q)read_data~array_data
1b
```

> > :warning: With writing a large table Arrow may raise **'Capacity error: Cannot write arrays larger than 2^31 - 1 in length**. Preferable [way](https://arrow.apache.org/docs/python/ipc.html) of serializing of such a table is dividing it into chunks providing `ARROW_CHUNK_ROWS` option.

Reviewer: ```suggestion

> :warning: **When writing a large table Arrow may raise 'Capacity error: Cannot write arrays larger than 2^31 - 1 in length**. 
>
> Preferable [way](https://arrow.apache.org/docs/python/ipc.html) of serializing of such a table is dividing it into chunks by specifying `ARROW_CHUNK_ROWS` option.

```

```q
table:([]col:2147483652#0x00)
options:(``ARROW_CHUNK_ROWS)!((::);214748365)
.arrowkdb.ipc.writeArrowFromTable["table.arrow";table;options]
```

> ### `ipc.writeArrowFromTable`

Reviewer: ```suggestion

### `ipc.writeArrowFromTable`
```

q)read_table~table
1b
```

> > :warning: With writing a large table Arrow may rize **'Capacity error: Cannot write arrays larger than 2^31 - 1 in length**. Preferable [way](https://arrow.apache.org/docs/python/ipc.html) of serializing of such a table is dividing it into chunks providing `ARROW_CHUNK_ROWS` option.

Reviewer: ```suggestion
> :warning: **When writing a large table Arrow may raise 'Capacity error: Cannot write arrays larger than 2^31 - 1 in length**. 
>
> Preferable [way](https://arrow.apache.org/docs/python/ipc.html) of serializing of such a table is dividing it into chunks by specifying `ARROW_CHUNK_ROWS` option.
```

q)read_table~table
1b
```

> > :warning: **When writing a large table Arrow may raise 'Capacity error: Cannot write arrays larger than 2^31 - 1 in length**. 

Reviewer: ```suggestion

> :warning: **When writing a large table Arrow may raise 'Capacity error: Cannot write arrays larger than 2^31 - 1 in length**. 
```

>

> > Preferable [way](https://arrow.apache.org/docs/python/ipc.html) of serializing of such a table is dividing it into chunks by specifying `ARROW_CHUNK_ROWS` option.

Reviewer: ```suggestion
> Preferable [way](https://arrow.apache.org/docs/python/ipc.html) of serializing of such a table is dividing it into chunks by specifying `ARROW_CHUNK_ROWS` option.

```

q)read_data~array_data
1b
```

> > :warning: With writing a large table Arrow may raise **'Capacity error: Cannot write arrays larger than 2^31 - 1 in length**. Preferable [way](https://arrow.apache.org/docs/python/ipc.html) of serializing of such a table is dividing it into chunks providing `ARROW_CHUNK_ROWS` option.

Reviewer: ```suggestion

> :warning: **When writing a large table Arrow may raise 'Capacity error: Cannot write arrays larger than 2^31 - 1 in length**. 
>
> Preferable [way](https://arrow.apache.org/docs/python/ipc.html) of serializing of such a table is dividing it into chunks by specifying `ARROW_CHUNK_ROWS` option.
```

q)new_table~table
1b
```

> > :warning: With writing a large table Arrow may raise **'Capacity error: Cannot write arrays larger than 2^31 - 1 in length**. Preferable [way](https://arrow.apache.org/docs/python/ipc.html) of serializing of such a table is dividing it into chunks providing `ARROW_CHUNK_ROWS` option.

Reviewer: ```suggestion

> :warning: **When writing a large table Arrow may raise 'Capacity error: Cannot write arrays larger than 2^31 - 1 in length**. 
>
> Preferable [way](https://arrow.apache.org/docs/python/ipc.html) of serializing of such a table is dividing it into chunks by specifying `ARROW_CHUNK_ROWS` option.

```

> ### `ipc.parseArrowSchema`

Reviewer: ```suggestion
```
```q
table:([]col:2147483652#0x00)
options:(``ARROW_CHUNK_ROWS)!((::);214748365)
serialized:.arrowkdb.ipc.serializeArrowFromTable["table.arrow";table;options]
```

### `ipc.parseArrowSchema`
```
